### PR TITLE
Added opx_finance optimism trades to dex_trades.sql

### PR DESCRIPTION
**Pull request by kaiblade**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

File: tests/dex/dex_trades_assert_freshness.sql
Comment (jeff-dude) [inline tests/dex/dex_trades_assert_freshness.sql:1]: you can revert code added to this file and omit from the PR

Comment (kaiblade) [reply tests/dex/dex_trades_assert_freshness.sql:1]: Strangely, without doing much, all checks have been approved, seems the issue got resolved by itself. I have marked this 
PR as ready for review.  Can you please check it? Thanks!

File: models/opx_finance/optimism/opx_finance_optimism_trades.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
+{{ config(tags=['dunesql'],
+    aliAS = alias('trades')
```
Comment (jeff-dude) [inline models/opx_finance/optimism/opx_finance_optimism_trades.sql:2]: minor: can you lowercase `alias` here? also, maybe move the comma to left side similar to below, to make this model config quick/easier to look at in future?

**Review by jeff-dude — CHANGES_REQUESTED** [review]
one minor note below, otherwise looks good!

File: models/opx_finance/optimism/opx_finance_optimism_trades.sql (hunk 1 above)
Comment (kaiblade) [reply models/opx_finance/optimism/opx_finance_optimism_trades.sql:2]: Ok

Comment (kaiblade) [reply models/opx_finance/optimism/opx_finance_optimism_trades.sql]: Done. Kindly recheck.

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**